apps/frontend/site/.env.example

```
@@ -16,4 +16,8 @@ PUBLIC_DISABLE_APP_CHECK=
 
 PUBLIC_MODE=
 PUBLIC_LOG_PERSIST_LEVEL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUBLIC_LOG_PERSIST_LEVEL key should go before the PUBLIC_MODE key

(UnorderedKey)